Doc 1671/archive v1 onboarding docs to preview as reference for v2 gap analysis (#607)
swan-io/docs: docs/preview/new-onboarding/legacy-onboarding/company/guide-update.mdx

---
title: Update a company onboarding
---

# Update a company onboarding

:::caution Deprecated

> This page documents the legacy onboarding API. It is preserved as a read-only reference. Refer to [account onboarding enhancements](../../../index.mdx) for the current v2 API.

Reviewer: Above we call say "This page documents the v1 onboarding API." - here its 'legacy' - find one way of referencing the original version and stick to it. Partners wont know what v1/legacy etc are.

Author: Agreed, applying. Aligning all instances to "previous onboarding API."

> This page documents the legacy onboarding API. It is preserved as a read-only reference. Refer to [account onboarding enhancements](../../../index.mdx) for the current v2 API.

Author: ```suggestion
This page documents the legacy onboarding API. It is preserved as a read-only reference. Refer to [account onboarding enhancements](../../../index.mdx) for the current onboarding API.
```

:::

Update company onboardings using the API.

:::caution Updating UBO fields

> Fields about an ultimate beneficial owner (UBO) can't be updated incrementally.

Reviewer: What does this mean exactly? 

e.g. 
- If you want to update UBO fields - you'll have to do all of them at once?
or
- You can't update UBO fields one by one - if you start doing it, you'll have to update all of them again?

Author: The original wording is accurate: if you need to update any UBO field, you must resubmit all UBO fields: they can't be updated incrementally. But I can make it clearer: "To update any UBO field, you must resubmit all UBO fields in full."

> Fields about an ultimate beneficial owner (UBO) can't be updated incrementally.

Author: ```suggestion
Fields about an ultimate beneficial owner (UBO) can't be updated incrementally. To update any UBO field, you must resubmit all UBO fields in full.
```

> If you need to update UBO information, **you must submit all of the fields again**.

Author: ```suggestion
Fields about an ultimate beneficial owner (UBO) can't be updated incrementally. If you need to update UBO information, **you must submit all of the fields again**.
```

:::

import UpdateOnboardingPrereqs from '../partials/_prereqs-update.mdx';

<UpdateOnboardingPrereqs />

## Guide {#api-guide}

> To test updating a company onboarding, **change the language**.

Reviewer: Are we testing this here or actually updating it? The steps don't reference the sanbox so unsure where the testing element comes in.

> To test updating a company onboarding, **change the language**.

Author: ```suggestion
To update a company onboarding, **change the language**.
```
